### ISSUE TITLE
Add alt-text tags to Create Boarding List Chart page

### ISSUE THREAD

**Issue by o-fish-wildaid** [WildAid/wildaid.github.io#68]
Add alt-text tags to the Create Boarding List Chart page. 

Page exists at: https://wildaid.github.io/charts/2020/06/12/Boardings-List.html
In the code at: _posts/build_charts/2020-06-12-Boardings-List.md

A brief guide to alt-text tags can be found in our style guide at https://wildaid.github.io/style/2020/10/02/Alt-Text.html

**Comment by RuchaYagnik** [comment]
Hi @o-fish-wildaid, I have gone through the issue requisite and would like to contribute to it. I request to assign this issue to me, Thank you.

**Comment by RuchaYagnik** [comment]
Hi @o-fish-wildaid, I have gone through the issue requisite and would like to contribute to it. I request to assign this issue to me, Thank you.

**Comment by Sheeri** [comment]
Hello! Thanks for expressing interest in this issue. Since you have been assigned issue #67 and haven't submitted a PR for it yet, we ask that you finish that issue first, before we assign another issue to you.

**Comment by RuchaYagnik** [comment]
Hi @Sheeri! as mentioned by you earlier, I have submitted a PR for issue #67. Can I be assigned this issue once the PR for issue #67 is reviewed ?

**Comment by Sheeri** [comment]
@RuchaYagnik great work on the other .... it was merged ! I have assigned this issue to you.... remember that if the alt text cannot be anything other than a repeat of the text just before it,  it's OK to do alt="". (The empty string)

**Comment by RuchaYagnik** [comment]
Will keep that in mind this time @Sheeri, Thanks for assigning this issue to me!
